src/lifecycle/render.js

```diff
@@ -1,18 +1,39 @@
 import { patchInner } from 'incremental-dom';
-import { shadowRoot as $shadowRoot } from '../util/symbols';
+import { connected as $connected, shadowRoot as $shadowRoot, state as $state } from '../util/symbols';
```

Comment:
yes, it broke some of the AtlasKit code the other week, so we decided to revert that change, as it is cosmetic and you can rename on import anyway.